concert/tests/unit/devices/test_camera.py

```diff
@@ -72,6 +72,11 @@ def grab():
         image = self.camera.grab()
         np.testing.assert_equal(image, grab()[:, ::-1])
 
+    def test_simulate(self):
+        self.assertTrue(np.any(self.camera._background - self.camera.grab()))
```

Comment:
Accessing the underscored properties is not very clean, but is it acceptable in tests?

Reply:
I'd say no. In my opinion, a unit test should test the same interface the user will use at the end. In this case, I would generate a known background in the `setUp` step, pass it to the camera and save it in the test case object to assert it later in your `test_simulate` step.

Reply:
Maan, you like to make me work, don't you?! Naah you are right, now it should be good.

Reply:
It's just one more line ;-)